sp21/group.yml

```diff
@@ -1,11 +1,14 @@
 # Check the Docs for details on editing this!
 name: "gbms"
-room: ""
+room: "Online"
 semester: "sp21"
 startdate: "2021-01-18T00:00:00"
```

Comment:
Do you have a meeting date/time? Otherwise it'll show on the website that you're meeting yesterday at 12AM.

Reply:
is this utc time zone?

Reply:
Yes, **but** that's handled on the website's side.